orchestrate.py

```diff
@@ -244,8 +256,10 @@ def main():
     handlers = [
         (r"/", LoadingHandler),
         (r"/spawn/?(/user/\w+(?:/.*)?)?", SpawnHandler),
+        (r"/spawn/((?:notebooks|tree)(?:/.*)?)", SpawnHandler),
```

Comment:
Not sure that if looks as nice if we use a query parameter. I was hoping that the initial URL looked clean and familiar to Notebook users.